tests/params/test_outputs.py

```diff
@@ -36,6 +34,11 @@ def change_test_dir(request, monkeypatch):
     monkeypatch.chdir(request.fspath.dirname)
 
 
+@pytest.fixture
```

Comment:
This fixes the ruff complaint about `array_equality_override`/`mock_response` not used but if I pass the `setup_array_equality_override` to the test function then the unit test fails so this probably is just a hack instead of a fix.

Not sure what is a proper way as I thought passing the `setup_array_equality_override` to the test function should have worked...

Reply:
There are many similar instances.

Reply:
https://github.com/astral-sh/ruff/issues/4046

Seems like we are not the only ones having this issue.

The suggested solution is moving the fixture definition to conftest.py it seems.

Reply:
Changed to use pytest_plugins in conftest.py